rllib/models/torch/encoders/vector.py

```diff
@@ -0,0 +1,129 @@
+from typing import TYPE_CHECKING, List
+
+import torch
+from torch import nn
+
+from ray.rllib.models.specs.specs_dict import ModelSpec
+from ray.rllib.models.torch.model import TorchModel
+from ray.rllib.models.utils import get_activation_fn
+from ray.rllib.utils.nested_dict import NestedDict
+
+if TYPE_CHECKING:
+    from ray.rllib.models.configs.encoder import VectorEncoderConfig
+
+
+def input_to_output_spec(
+    input_spec: ModelSpec, output_key: str, feature_dims: List[int]
+) -> ModelSpec:
+    """Convert an input spec to an output spec by replacing the input dims and
+    key with the output dims and output key.
+
+    E.g.
+    input_to_output_spec(
```

Comment:
Can we clarify that the input "a, b, c, d" dimensions have nothing to do with the output ones (which have the same name in this example)?

Basically, use different letters for the output?

```
should return
ModelSpec({"foo": "w, x, y, z"}, z=2)
```